### PR TITLE
Fetch fee multipliers from Kalshi get-series-fee-changes endpoint

### LINKED ISSUE #76: Fetch fee multipliers from Kalshi get-series-fee-changes endpoint
## Context
Split from #39. The fee multipliers (0.07 taker, 0.0175 maker) are currently hardcoded. Kalshi provides a `get-series-fee-changes` endpoint with per-series multipliers that may differ.

## Task
1. Add API function to query `get-series-fee-changes` endpoint
2. Cache multipliers with TTL per series
3. Thread series-specific multipliers through `fee_for_order` calls

## Priority
Medium — current hardcoded values are correct for most markets.

### PR DESCRIPTION
**Pull request by allan-mobley-jr**
## Summary
- Add `FeeMultipliers` dataclass and per-series fee cache with 1-hour TTL, fetched from Kalshi's `GET /series/fee_changes` endpoint
- Thread `FeeMultipliers` through the full trading pipeline: fee calculation, Kelly sizing, scoring, validation, and paper fill simulation
- Add `series_ticker` field to Market model and parse from API responses
- Graceful fallback to hardcoded defaults (0.07 taker / 0.0175 maker) on API failure, with bounds validation and future-scheduled change filtering

Closes #76

## Test plan
- [x] 430 unit tests pass (14 new tests covering cache behavior, multiplier threading, and edge cases)
- [x] All existing tests pass unchanged (backward-compatible via defaults)
- [x] Lint clean on all changed files
- [x] Verified: API failure falls back gracefully to defaults
- [x] Verified: invalid/out-of-range multipliers are skipped with warnings
- [x] Verified: future-scheduled fee changes are filtered out

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/gimmes/strategy/fee_cache.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    for item in records:
+        series = item.get("series_ticker", "")
+        if not series:
+            continue
+
+        # Skip future-scheduled changes that haven't taken effect yet
+        scheduled = item.get("scheduled_ts")
+        if scheduled:
+            try:
+                ts = datetime.fromisoformat(scheduled)
```
[the remark this answers is not in the document]
Comment (allan-mobley-jr) [reply src/gimmes/strategy/fee_cache.py]: Fixed in 99c097f. Extracted `_parse_scheduled_ts` helper that normalizes `Z` to `+00:00` before calling `fromisoformat()`, ensuring Python 3.11 compatibility.
